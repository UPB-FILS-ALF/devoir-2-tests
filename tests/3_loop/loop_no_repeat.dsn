& LOOP no REPEAT

LOOP: 2
CIRCLE: 100 100 100
CIRCLE: 50 100 100 