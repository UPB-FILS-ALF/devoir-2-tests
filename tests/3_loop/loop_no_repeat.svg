<?xml version="1.0" encoding="UTF-8" ?>
<svg xmlns="http://www.w3.org/2000/svg" version="1.1" width="1024" height="768">
	<!-- loop 2: 2 -->
	<!-- loop #0 -->
		<circle cx="100" cy="100" r="100" stroke="rgb(0,0,0)"  stroke-width="1" fill="rgb(255,255,255)"/>
		<circle cx="50" cy="100" r="100" stroke="rgb(0,0,0)"  stroke-width="1" fill="rgb(255,255,255)"/>
</svg>
