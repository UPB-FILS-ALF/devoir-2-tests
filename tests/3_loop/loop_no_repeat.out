LINE 3 HAS ERROR: LOOP with no REPEAT
